firmware/application/clock_manager.hpp

```diff
@@ -93,9 +97,8 @@ class ClockManager {
 
     uint32_t measure_gp_clkin_frequency();
 
-    ReferenceSource detect_reference_source();
-    Reference choose_reference();
-    bool loss_of_signal();
+    ReferenceSource detect_reference_source(bool trueCheck = false);
```

Comment:
Nit: what does ‘trueCheck’ mean? Is there another way to name it?

Reply:
The current check is not the true way to actually detect the TCXO, but if you enable TrueCheck, then it uses the method to truly detect the TCXO

Reply:
Maybe something like ‘useRealCheck’ or something along those lines. I’d avoid “true” just because it’s a bool.

Reply:
Yeah fair enough, I'll update that now